process.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>

```

Comment:
we need the part01/index.php file for this one to work. it contains a form that submits its post array to this file.